data/virt_autotest/guest_params_xml_files/sles_15_sp5_64_xen_hvm_uefi_with_power_management_x86_64.xml

```diff
@@ -84,7 +84,7 @@
     <guest_autoconsole>text</guest_autoconsole>
     <guest_noreboot></guest_noreboot>
     <guest_default_target>graphical</guest_default_target>
-    <guest_do_registration>true</guest_do_registration>
+    <guest_do_registration>false</guest_do_registration>
```

Comment:
I remember the registration bug is fixed, right? Can we switch on this?

Reply:
Done. Verification updated.